drivers/phy/qualcomm/phy-qcom-qusb2.c

```diff
@@ -810,6 +851,12 @@ static const struct phy_ops qusb2_phy_gen_ops = {
 
 static const struct of_device_id qusb2_phy_of_match_table[] = {
 	{
+		.compatible	= "qcom,ipq6018-qusb2-phy",
+		.data		= &ipq6018_phy_cfg,
+	}, {
+		.compatible	= "qcom,ipq8074-qusb2-phy",
+		.data		= &msm8996_phy_cfg,
```

Comment:
This is not part of upstream commit `2cfbe6765b7a`. Please remove.

Reply:
Hi Baruch
This was added with the patch 0001-phy-qcom-qusb2-add-QUSB2-support-for-IPQ6018.patch
Do you want me to edit this out?

Reply:
These line are not part of the original commit. They were probably added while resolving the merge conflict. See the original commit here:

  https://git.kernel.org/linus/2cfbe6765b7a

Reply:
maybe I mess something but when I do cat ~/0001-phy-qcom-qusb2-add-QUSB2-support-for-IPQ6018.patch
I see:

 static const struct of_device_id qusb2_phy_of_match_table[] = {
        {
+               .compatible     = "qcom,ipq6018-qusb2-phy",
+               .data           = &ipq6018_phy_cfg,
+       }, {
+               .compatible     = "qcom,ipq8074-qusb2-phy",
+               .data           = &msm8996_phy_cfg,
+       }, {
                .compatible     = "qcom,msm8996-qusb2-phy",
                .data           = &msm8996_phy_cfg,
        }, {

doesn't is means that those lines are part of the original patch?

Reply:
I don't know where this patch file is coming from. These lines are not part of the original commit I linked to above, and also not part of the patch on the mailing list the was applied in this commit:

  https://lore.kernel.org/r/b8c22dddf1f70d89e135fe1ae705ddc68e295ebb.1611756920.git.baruch@tkos.co.il

Just remove these lines from the commit.

Reply:
Hi Baruch.
Apparently those 2 lines makes a difference . without them the nfs does not work (stuck). I just tried it.

Reply:
The last commit in the current usb_support branch is missing this hank entirely (all 5 lines). You only need to remove the last two lines. Without the `compatible` line nothing works.